.changeset/khaki-lies-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': minor
```

Comment:
Should be `patch` (when on 0.x versions, that means nonbreaking)

Reply:
done